polynote-frontend/style/styles.less

```diff
@@ -1495,17 +1495,25 @@ button.inspect.icon-button {
     grid-area: head;
     position: relative;
     user-select: none;
+    display: flex; 
+    align-items: center; 
+    min-width: 0; 
+    overflow: hidden;
+    height: 1em;
+
     .left-buttons, .right-buttons {
-      position: absolute;
-      top: .1em;
-      bottom: .1em;
       line-height: 1.8em;
     }
 
     .right-buttons {
-      right: 0;
       text-align: right;
-      padding-right: .25em;
```

Comment:
Same here. I think making this `display: flex` on line 1498 makes sense, but we do lose these area paddings, no?